electron/main.js

```diff
@@ -219,6 +232,14 @@ app.on("ready", async () => {
     await utils.exportSave(window);
   } else {
     const window = await startWindow(process.argv.includes("--no-scripts"));
+    app.on("second-instance", (event, commandLine, workingDirectory, additionalData) => {
```

Comment:
Like the example in the docs, let's put this right up next to the lock checking code. Keeps it easier to think about related stuff together.